dashboard/src/assets/styles/resources.scss

```diff
@@ -15,6 +15,7 @@ $background-emphasis-color: #fafafa;
 
 $border-color: #dadee2;
 $border-strong-color: #cccccc;
+$border-light-color: #f2f2f2;
```

Comment:
If we're going to use the same value as `$background-content-color`, let's use this instead:

```sass
$border-light-color: $background-content-color;
```